src/pages/ecosystem.js

```diff
@@ -133,6 +133,25 @@ export default function Home() {
           </div>
         </Section>
 
+        <Section>
+          <div className="grid grid-cols-1 lg:grid-cols-2 gap-8 lg:gap-14 mb-44">
+            <div className="">
+              <div className="w-full mb-8 bg-substrateGray-light dark:bg-substrateGray-darkest">
+                <StaticImage src="../images/png/square-one/square-one.png" alt="Square One" className="rounded-md" />
+              </div>
+            </div>
+            <div className="">
+              <h2 className="mb-6 text-4xl md:text-5xl lg:text-6xl font-extrabold">Square One</h2>
+              <h3 className="mb-6 text-2xl md:text-3xl font-medium lg:pr-4">Your ecosystem guide</h3>
+              <p className="md:text-lg font-medium">
+                Square One is your custom-tailored guide to the opportunities and initiatives available in the
+                ecosystem. Explore the resources and initiatives below or get in touch for hands-on support.
```

Comment:
Should probably say "Explore resources and initiatives or get in touch for hands-on support.", since the use of "below" isn't really correct until you're on the Square One page.